src/SFA.DAS.AssessorService.Application.Api/Validators/EpaOrganisationValidator.cs

```diff
@@ -513,8 +513,10 @@ public ValidationResponse ValidatorUpdateEpaOrganisationContactRequest(UpdateEpa
             RunValidationCheckAndAppendAnyError("OrganisationId", CheckIfOrganisationNotFound(request.OrganisationId), validationResult, ValidationStatusCode.NotFound);
             if (!validationResult.IsValid) return validationResult;
 
-            RunValidationCheckAndAppendAnyError("OrganisationId", CheckIfOrganisationStandardAlreadyExists(request.OrganisationId, request.StandardCode), validationResult, ValidationStatusCode.AlreadyExists);
-            if (!validationResult.IsValid) return validationResult;
+            //  SV-658 / SV-659 Now that we have versions, it is possible to register a version where the organisation+standard already exists.
+            //  So this validation is now redundant.
```

Comment:
Should this validation be deleted as it's no longer required, or should it be upgraded to now throw an error if the standard versions being sent in already exist and are approved for?

Reply:
After talking with Al, I think we should re-instate and improve the rule to be

If we're adding a new full standard, and it exists, and it's not withdrawn -> failure
if it is withdrawn -> success

if adding a version, and the version already exists and not withdrawn -> failure
if it is withdrawn -> success

Overall it's not great as once the error is returned, the admin person can't do anything about it but it at least maintains existing behaviour with the versioning upgrade.